File: viajes/src/gestorAplicacion/hangar/Silla.java
package gestorAplicacion.hangar;
import gestorAplicacion.adminVuelos.*;
import java.io.Serializable;
public class Silla implements Serializable {

	private Clase clase;
	private int numero_de_silla;
	private Pasajero pasajero;
	private boolean estaDisponible = true;
	private Ubicacion ubicacion;
	
	
	public Silla(Clase clase, int numero_de_silla,Ubicacion ubicacion) {
		this.clase = clase;
		this.numero_de_silla = numero_de_silla;
		this.ubicacion = ubicacion;
	}
	
	//GETTERS Y SETTERS

	public Clase getClase() {
		return clase;
	}
	public void setClase(Clase clase) {
		this.clase = clase;
	}
	public int getNumero_de_silla() {
		return numero_de_silla;
	}
	public void setNumero_de_silla(int numero_de_silla) {
		this.numero_de_silla = numero_de_silla;
	}
	public Pasajero getPasajero() {
		return pasajero;
	}
	public void setPasajero(Pasajero pasajero) {
		this.pasajero = pasajero;
	}
	public boolean isEstado() {
		return estaDisponible;
	}
	public void setEstado(boolean estado) {
		this.estaDisponible = estado;
	}
	public Ubicacion getUbicacion() {
		return ubicacion;
	}
	public void setUbicacion(Ubicacion ubicacion) {
		this.ubicacion = ubicacion;
	}
	
	
}

File: carpeta/gestorAplicacion/hangar/Silla.py
from gestorAplicacion.adminVuelos import *

class Silla:
    def __init__(self, clase, numero_de_silla, ubicacion):
        self._pasajero = None
        #instance fields found by Java to Python Converter:
        self._estaDisponible = True
        self._clase = clase
        self._numero_de_silla = numero_de_silla
        self._ubicacion = ubicacion

    #GETTERS Y SETTERS

    def getClase(self):
        return self._clase

    def setClase(self, clase):
        self._clase = clase

    def getNumero_de_silla(self):
        return self._numero_de_silla

    def setNumero_de_silla(self, numero_de_silla):
        self._numero_de_silla = numero_de_silla

    def getPasajero(self):
        return self._pasajero

    def setPasajero(self, pasajero):
        self._pasajero = pasajero

    def isEstado(self):
        return self._estaDisponible

    def setEstado(self, estado):
        self._estaDisponible = estado

    def getUbicacion(self):
        return self._ubicacion
        
    def setUbicacion(self, ubicacion):
        self._ubicacion = ubicacion